hw4/tests/task2_test.go

```diff
@@ -0,0 +1,23 @@
+package tests
+
+import (
+	"math"
+	"testing"
+
+	"github.com/COOLizh/epam/epamGolangTasks/hw4/tasks"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCalculation(t *testing.T) {
+	assert := assert.New(t)
+	var s tasks.Figure
+	var c tasks.Figure
+	for i := 1; i <= 10; i++ {
```

Comment:
As the first glance it seems like a good idea for test case, but in practice it is not recommended.
Not a blocker here though.